mne/evoked.py

```diff
@@ -727,6 +727,42 @@ def plot_white(self, noise_cov, show=True):
         return _plot_evoked_white(self, noise_cov=noise_cov, scalings=None,
                                   rank=None, show=show)
 
+    def animate(self, ch_type, frames=5, interval=100, butterfly=False,
+                blit=True, show=True):
+        """Make animation of evoked data as topomap timeseries.
+
+        Parameters
+        ----------
+        ch_type : str
+            Channel type to plot. Accepted data types: 'mag', 'grad', 'eeg'.
+        frames : int | list of ints
+            If int, the number of frames to animate. If list of ints, the
+            indices to plot in the animation. Defaults to 5.
+        interval : int
+            The time interval before drawing a new frame as milliseconds.
+            Defaults to 100.
```

Comment:
I think of it as the difference between aesthetics and standardized
scientific display. If I want to show someone data from an experiment, it's
easier to say that data is presented X times slower than real time, and 10x
seems like a reasonable default. I can't see how e.g. a fixed 60 Hz will
make data easily understandable for data recorded at 100 Hz as well as it
will for data recorded at 2000 Hz. One will be too fast, and the other too
slow. But they will both be equally "smooth" during presentation, though.
To me that smoothness tradeoff is not worth the understandability tradeoff.
But if others don't see it that way I'm fine with it.